src/frontends/tensorflow/src/op/while.cpp

```diff
@@ -42,9 +42,9 @@ OutputVector translate_while_op(const NodeContext& node) {
         body_model,
         "[TensorFlow Frontend] Internal error or incorrect input model. Cannot find body graph with name " + body_type);
 
-    auto loop = create_loop_for_tf_while(node.get_name(), body_model, cond_model, ov_inputs);
-    set_node_name(node.get_name(), loop);
-    return loop->outputs();
+    auto loop_outputs = create_loop_for_tf_while(node.get_name(), body_model, cond_model, ov_inputs);
+    set_node_name(node.get_name(), loop_outputs[0].get_node_shared_ptr());
```

Comment:
maybe it's better to return tuple from create_loop_for_tf_while? [loop, actual_outputs]?